src/main/resources/db/migration/V048__alter_agreement_add_organization_name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE agreement ADD COLUMN organization_name VARCHAR(100) NOT NULL;
```

Comment:
You should add a DEFAULT value, remember that we are going to apply this ALTER to an already existing database that won't have a value for this column and NOT NULL without a DEFAULT is going to break.